orienteer-core/src/main/java/org/orienteer/core/method/Filter.java

```diff
@@ -0,0 +1,16 @@
+package org.orienteer.core.method;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+/**
+ * 
+ * Filter annotation for {@link Method} annotation 
+ * Method will display only if all filters passed
+ *
+ */
+
+@Retention(RetentionPolicy.RUNTIME)
+public @interface Filter{
```

Comment:
I understood what was concerning me with using "Filter" word. It's being used in J2EE for HTTP filters. So it's confusing" lets think about alternatives.